cmd/notary/tuf_test.go

```diff
@@ -33,7 +44,21 @@ func TestTokenAuth200Status(t *testing.T) {
 	s := httptest.NewServer(http.HandlerFunc(NotAuthorizedTestHandler))
 	defer s.Close()
 
-	auth, err := tokenAuth(s.URL, baseTransport, gun, readOnly)
+	auth, err := tokenAuth(s.URL, baseTransport, gun, readOnly, false)
+	require.NoError(t, err)
+	require.NotNil(t, auth)
+}
+
+func TestAdminTokenAuth200Status(t *testing.T) {
```

Comment:
Totally non-blocking: good to have these here, but I'm not sure our existing auth tests actually do anything either other than reflect the 401 unauthorized handler. We don't actually test which permissions are asked for - I'm wondering if it's worth writing a mock jwt authorizer, so we can ensure that notary is asking for the correct permissions.  However, a lot of that stuff is tested in the distribution library we depend on too.  

What do you think?  I can file an issue if that plumbing would be a useful thing for us to test.